README.md

```diff
@@ -1 +1,274 @@
+
+- GitHub: [@oscar-byte15](https://github.com/oscar-byte15)
+- Twitter: [@oscardlacruz15](https://twitter.com/oscardlacruz15)
+- LinkedIn: [Oscar De la Cruz](https://www.linkedin.com/in/oscar-daniel-de-la-cruz-jara/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭  Future Features:
+- [ ] **[more personal projects]**
+- [ ] **[more functions to see the information]**
+- [ ] **[share content on social media]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, you cand send me a message through any social media that I own
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio Project
+
+In this project, I set up my portfolio repository. My goal here was to set up and customize my portfolio for mobile and desktop versions.
+
+
+## Live Demo
+
+Not available
+
+## Built With
+
+- HTML and CSS
+
+To get a local copy up and running follow these simple steps:
+
+1. Go to the [repository page](https://github.com/oscar-byte15/portfolio_finish_mobile.git).
+2. Press the "Code" button and copy the link.
+3. Clone it using git command `git clone <link>`.
+
+## Learning Objectives
+
+- Understand how to parse a Figma design to create a UI.
+- Flexbox to place elements in the page.
+- Build a personal portfolio site.
+- Use images and backgrounds to enhance the look of the website.
+
+## Authors
+
+👤 **Oscar De la Cruz**
+
+- GitHub: [@oscar-byte15](https://github.com/oscar-byte15)
+
+
+## 🤝 Contributions
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/oscar-byte15).
+
+## Acknowledgments
+
+- Thanks to Microverse
+
+## 📝 License
+
+This project is [MIT](LICENSE) licensed.
```

Comment:
I do not think this section of the readme is relevant, since most of the sections here have been covered previously😉  I suggest you kindly take it away😎

Reply:
ok, I have done it!